tests/test_linear_referencing.py

```diff
@@ -6,14 +6,13 @@
 
 
 class LinearReferencingTestCase(unittest.TestCase):
-
     def setUp(self):
         self.point = Point(1, 1)
-        self.line1 = LineString(([0, 0], [2, 0]))
-        self.line2 = LineString(([3, 0], [3, 6]))
-        self.multiline = MultiLineString([
-            list(self.line1.coords), list(self.line2.coords)
-        ])
+        self.line1 = LineString([(0, 0), (2, 0)])
```

Comment:
Lines 11-12 are the only intentional changes in this file; everything else is autoformatting